src/views/Log.vue

```diff
@@ -474,6 +460,18 @@ export default {
         }
       }
       return this.workflowId
+    },
+    workflowStatus () {
+      if (this.workflows[0].node.stateTotals.failed) {
+        return 'failed'
+      }
+      if (this.workflows[0].node.stateTotals.submitted | this.workflows[0].node.stateTotals.running | this.workflows[0].node.stateTotals.succeeded) {
+        return 'succeeded'
+      }
+      if (this.workflows[0].node.stateTotals['submit-failed']) {
+        return 'submit-failed'
+      }
+      return 0
```

Comment:
The workflow's `stateTotals` tells us how many tasks are in each state (e.g. if there are *any* failed tasks in the workflow), not the state of the particular job that we are requesting the log file for.


To get the job status, I think you want something along these lines:

```js
if (this.jobLog) {
  // we are looking at a job log => pick the appropriate log file based on its state

  // get the job from the store
  this.$workflowService.$index[this.id]

  return ...
} else {
   // we are viewing the workflow log => always default to the latest log file

  return ...
}
```

Reply:
Dammit, good point, we could use an offline query, but I'm not sure if we can retrieve the status field without making changes to the backend.

Reply:
@markgrahamdawson From discussing today, we'll settle on a live query, like this:
```gql
query JobState($id: ID!) {
  job (id: $id) {
    state
  }
}
```
(As I mentioned, we don't need the query if the `file` in `intialOptions` is pre-populated as in https://github.com/cylc/cylc-ui/compare/master...MetRonnie:cylc-ui:menu-log-file)